test/unit/expressions.spec.ts

```diff
@@ -48,6 +48,14 @@ test.each(["a+=b", "a-=b", "a*=b", "a/=b", "a%=b", "a**=b"])("Binary expressions
 const supportedInAll = ["~a", "a&b", "a&=b", "a|b", "a|=b", "a^b", "a^=b", "a<<b", "a<<=b", "a>>>b", "a>>>=b"];
 const unsupportedIn53And54 = ["a>>b", "a>>=b"];
 const allBinaryOperators = [...supportedInAll, ...unsupportedIn53And54];
+test.each(allBinaryOperators)("Bitop [5.0] (%p)", input => {
```

Comment:
This test is good, but I'm still missing a `testEachVersion` test in this file testing `%` modulo, since that is now version-specific.

Reply:
Added a special test for `%` with fac16d27576b541b3f5d3b248999b995e373b8fa. (Did I do that right?)